moveit_setup_assistant/templates/moveit_config_pkg_template/launch/sensor_manager.launch.xml

```diff
@@ -2,6 +2,9 @@
 
   <!-- This file makes it easy to include the settings for sensor managers -->
 
+  <!-- Params for 3D sensors config -->
+  <rosparam command="load" file="$(find [GENERATED_PACKAGE_NAME])/config/sensors_3d.yaml" />
```

Comment:
Late, but what was the rationale for not using the "robot specific sensor manager" `.launch.xml` infrastructure used further on in this file to load the `.yaml`?

This seems to have deviated from how this worked previously, and seems to make `[ROBOT_NAME]_moveit_sensor_manager.launch.xml` redundant (but that file is still generated by the Setup Assistant).